Gemfile

```diff
@@ -21,6 +21,7 @@ group :test do
   gem 'factory_bot_rails'
   gem 'faker'
   gem 'rspec-rails'
+  gem 'rubocop-rails'
```

Comment:
I believe that rubocop-rails is already included by boost-styles. It might not be included by the rubocop though?

Reply:
Oh nice, I will remove it :D